frappe/patches/v14_0/different_encryption_key.py

```diff
@@ -0,0 +1,17 @@
+import pathlib
+
+import frappe
+from frappe.installer import update_site_config
+from frappe.utils.backups import BACKUP_ENCRYPTION_CONFIG_KEY, get_backup_path
+from frappe.utils.password import get_encryption_key
+
+
+def execute():
+	if frappe.conf.get(BACKUP_ENCRYPTION_CONFIG_KEY):
+		return
+
+	backup_path = pathlib.Path(get_backup_path())
+	encrypted_backups_present = bool(list(backup_path.glob("*-enc*")))
+
+	if encrypted_backups_present:
+		update_site_config(BACKUP_ENCRYPTION_CONFIG_KEY, get_encryption_key())
```

Comment:
`get_encryption_key` isn't needed here IMO.. it's definition may change in future, but we know that `frappe.conf.encryption_key` exists (since we have backup) and was the older key.


```suggestion
		update_site_config(BACKUP_ENCRYPTION_CONFIG_KEY, frappe.conf.encryption_key)
```

Reply:
Done